packages/web-app-files/src/components/SideBar/Shares/Collaborators/NewCollaborator.vue

```diff
@@ -58,6 +59,7 @@
           v-else
           id="files-collaborators-collaborator-save-new-share-button"
           key="new-collaborator-save-button"
+          data-testid="new-collaborator-share-btn"
```

Comment:
Theoretically could be removed in favor of the already exisiting `id` on that button

Reply:
I think you can solve it differently, but using `getById` in jest is nice. Happy to keep the `data-testid`s.